keyboards/handwired/oled25/rules.mk

```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
```

Comment:
Should only be enabled at the keymap level (also never within the default keymap). The convention would be to add a `via` keymap.
```suggestion
```

Reply:
added this change and a separate via keymap